client/shared-settings/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { combineReducers } from 'redux';
+import SharedSettingsRootView from './views';
+import reducer from 'lib/form-base/reducer';
+// from calypso
+import notices from 'state/notices/reducer';
+
+export default ( { formData, formMeta, storeOptions } ) => ( {
+	getReducer() {
+		return combineReducers( {
+			form: reducer,
+			notices,
+		} );
+	},
+
+	getHotReducer() {
+		return combineReducers( {
+			form: reducer,
```

Comment:
You need to `require()` the reducer again, or this hot-reloading won't work (anyways, I think our Webpack hot-reloading is mostly broken).